plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -0,0 +1,147 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import * as theia from '@theia/plugin';
+import { RemoteSshKeyManager, SshKeyManager, CheService } from './node/ssh-key-manager';
+import WorkspaceClient from '@eclipse-che/workspace-client';
```

Comment:
why do we need workspace-client ?
AFAIK @eclipse-che/plugin should provide the access to ssh no ?